ReactAndroid/src/main/java/com/facebook/react/views/text/ReactFontManager.java

```diff
@@ -60,6 +65,13 @@ public static ReactFontManager getInstance() {
       mFontCache.put(fontFamilyName, fontFamily);
     }
 
+    if(mHasCustomTypeface && mCustomTypefaceCache.containsKey(fontFamilyName)) {
+      return Typeface.create(
+        mCustomTypefaceCache.get(fontFamilyName),
+        style
+      );
+    }
+
     Typeface typeface = fontFamily.getTypeface(style);
     if (typeface == null) {
       typeface = createTypeface(fontFamilyName, style, assetManager);
```

Comment:
Is there value caching invidual styled Typeface (normal/italic/bold/boldItalic) instances here like it's done for fonts in assets? Besides that all LGTM.

Reply:
to use styles or weights you have to define fonts using XML.